.circleci/config.yml

```diff
@@ -0,0 +1,132 @@
+          name: Instal coveralls and gulp globally
+          command: sudo yarn global add coveralls gulp
+
+      # css-components
+      - restore_cache:
+          keys:
+            - v1-css-components-cache-{{ .Branch }}-{{ checksum "css-components/yarn.lock" }}
+            - v1-css-components-cache-{{ .Branch }}
+            - v1-css-components-cache-
+      - run:
+          name: Install css-components dependencies
+          command: |
+            cd css-components
+            yarn install --pure-lockfile
+      - save_cache:
+          key: v1-css-components-cache-{{ .Branch }}-{{ checksum "css-components/yarn.lock" }}
+          paths:
+            - ./css-components/node_modules
+
+      # OnsenUI
+      - restore_cache:
+          keys:
+            - v1-onsenui-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
+            - v1-onsenui-cache-{{ .Branch }}-
+            - v1-onsenui-cache-
+      - run:
+          name: Install dependencies
+          command: yarn install --pure-lockfile
+      - save_cache:
+          key: v1-onsenui-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: gulp build
+          command: gulp build
+
+      # bindings/angular2
+      - restore_cache:
+          keys:
+            - v1-angular2-cache-{{ .Branch }}-{{ checksum "bindings/angular2/yarn.lock" }}
+            - v1-angular2-cache-{{ .Branch }}-
+            - v1-angular2-cache-
+      - run:
+          name: Install Angular 2 dependencies
+          command: |
+            cd bindings/angular2
+            yarn install --pure-lockfile
+      - save_cache:
+          key: v1-angular2-cache-{{ .Branch }}-{{ checksum "bindings/angular2/yarn.lock" }}
+          paths:
+            - ./bindings/angular2/node_modules
+
+      # bindings/react
+      - restore_cache:
+          keys:
+            - v1-react-cache-{{ .Branch }}-{{ checksum "bindings/react/yarn.lock" }}
+            - v1-react-cache-{{ .Branch }}-
+            - v1-react-cache-
+      - run:
+          name: Install React dependencies
+          command: |
+            cd bindings/react
+            yarn install --pure-lockfile
+      - save_cache:
+          key: v1-react-cache-{{ .Branch }}-{{ checksum "bindings/react/yarn.lock" }}
+          paths:
+            - ./bindings/react/node_modules
+
+      # bindings/vue
+      - restore_cache:
+          keys:
+            - v1-vue-cache-{{ .Branch }}-{{ checksum "bindings/vue/yarn.lock" }}
+            - v1-vue-cache-{{ .Branch }}-
+            - v1-vue-cache-
+      - run:
+          name: Install Vue dependencies
+          command: |
+            cd bindings/vue
+            yarn install --pure-lockfile
+      - save_cache:
+          key: v1-vue-cache-{{ .Branch }}-{{ checksum "bindings/vue/yarn.lock" }}
+          paths:
+            - ./bindings/vue/node_modules
+
+      - run:
+          name: Run OnsenUI Tests
+          command: npm run test:skip-build
```

Comment:
Hi @serima, thank you for your great PR!

I think replacing this line with `gulp test --disable-warnings --skip-build --browsers local_chrome_headless` works well in this case.
`--browsers local_chrome_headless` option suppresses the unit testing on Safari in SauceLabs.

Reply:
Thank you. I will try it!